bin/install-app.sh

```diff
@@ -0,0 +1,45 @@
+#!/bin/sh
+# This program and the accompanying materials are
+# made available under the terms of the Eclipse Public License v2.0 which accompanies
+# this distribution, and is available at https://www.eclipse.org/legal/epl-v20.html
+# 
+# SPDX-License-Identifier: EPL-2.0
+# 
+# Copyright Contributors to the Zowe Project.
+if [ $# -eq 0 ]
+    then
+    echo "Usage: $0 AppPath"
+    exit 1
+fi
+utils_path=$(pwd)/../../zlux-server-framework/utils
```

Comment:
This requires that the current directory is zlux-app-server/bin.

What is someone runs it differently, e.g. they add  zlux-app-server/bin to PATH and just say install-app.sh? Or they cd into the plugin dir and say $ZOWE_APP_SERVER/bin/install-app.sh? Do we want to support it?

Reply:
We do want to support it, I just didn't yet. Do you have a suggestion? I'm still learning shell scripting.